### PR TITLE
Backport some fixes for building the release binaries

### PR DESCRIPTION
**Pull request by tstellar**

### CODE REVIEW

File: llvm/CMakeLists.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@ if(NOT DEFINED LLVM_VERSION_MINOR)
   set(LLVM_VERSION_MINOR 1)
 endif()
 if(NOT DEFINED LLVM_VERSION_PATCH)
-  set(LLVM_VERSION_PATCH 5)
+  set(LLVM_VERSION_PATCH 6)
```
Comment (boomanaiden154) [inline llvm/CMakeLists.txt:25]: Is this supposed to be included in this PR?

File: llvm/utils/lit/lit/__init__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 __author__ = "Daniel Dunbar"
 __email__ = "daniel@minormatter.com"
-__versioninfo__ = (18, 1, 5)
+__versioninfo__ = (18, 1, 6)
```
Comment (boomanaiden154) [inline llvm/utils/lit/lit/__init__.py:5]: Ditto here?

**Review by boomanaiden154 — APPROVED** [review]
Seems reasonable enough to me.

File: llvm/CMakeLists.txt (hunk 1 above)
Comment (tstellar) [reply llvm/CMakeLists.txt:25]: I just merged this commit in another PR.

**Pull request merged**